README.md

```diff
@@ -220,13 +220,14 @@ register whose address is given in MSRIndex column. For example, in the case of
 latency threshold value to write into the MSR defined in MSRIndex (0x3F6).
 
 ### CollectPEBSRecord
-Applies to processors that support both precise and non-precise events in Processor Event Based Sampling, such as Goldmont.
+Applies to processors that support both precise and non-precise events in **Processor** Event Based Sampling, such as Goldmont.
 
 0. The event cannot be programmed to collect a PEBS record.
 1. The event may be programmed to collect a PEBS record, but caution is advised. For instance, PEBS collection of this event may consume limited PEBS resources whereas interrupt-based sampling may be sufficient for the usage model.
 2. The event may programmed to collect a PEBS record, and due to the nature of the event, PEBS collection may be preferred. For instance,
```

Comment:
Missing "be" as the 4th word in sentence. Should read "The event may be programmed..."

Reply:
Thanks Fernando. Fixed in https://github.com/intel/perfmon/pull/113/commits/5994a9b1eb5d16ca92a3c7e10bae5a3beba1366f .